tests/core/test_broker.py

```diff
@@ -409,3 +411,12 @@ def test_pika_event_broker_configure_url(
     broker = PikaEventBroker(host=host, username=username, password=password)
     url = broker._configure_url()
     assert url == expected_url
+
+
+def test_celery_from_endpoint_config():
+    cfg = read_endpoint_config(
```

Comment:
Although we do have several YAML test configs from broker and other parts of Rasa we plan to move away from them as chasing them down is not as fast as having these as pytest fixtures.
Suggestion is to change this to:
```python
@pytest.fixture
def celery_endpoint_config_string() -> str:
    return """
        event_broker:
          type: celery
          broker_url: rediss://:password@localhost:6379/0
          task_name: tasks.event_logger
          countdown: 2
          priority: 1
    """

@pytest.fixture
def celery_endpoint_config(celery_endpoint_config_string: str) -> EndpointConfig:
   yaml_content = read_yaml(celery_endpoint_config_string)
   return EndpointConfig.from_dict(yaml_content)
   
 def test_celery_from_endpoint_config(celery_endpoint_config: EndpointConfig):
   actual = await EventBroker.create(celery_endpoint_config)

   assert isinstance(actual, CeleryEventBroker)
```